packages/@sanity/core/src/commands/dataset/copyDatasetCommand.js

```diff
@@ -54,6 +57,30 @@ const progress = (url) => {
   })
 }
 
+const followProgress = (jobId, client, output) => {
+  const spinner = output
+    .spinner({
+      text: `~0% done.`,
+    })
+    .start()
+
+  const listenUrl = client.getUrl(`jobs/${jobId}/listen`)
+
+  debug(`Listening on ${listenUrl}`)
+
+  progress(listenUrl).subscribe({
+    next: (event) => {
+      spinner.text = `~${event.progress}% done.`
```

Comment:
Use `{event.progress ? event.progress : 0}` ? Safer to add a field check than saying `~undefined% done.`, or just cry out loud saying can not process event.

Reply:
Makes sense! 🙂 Added in 065f86eb8